### PR TITLE
PATCH: update an article by article_id

### PR DESCRIPTION
**Pull request by ksskol**
I made some changes to the previous task (post) in the test file after receiving feedback. Also completed Task 8.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -222,16 +245,64 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 101,
```
Comment (roisinoreilly) [inline __tests__/app.test.js:277]: it would be good (but optional) to check the votes before the patch request and make sure they were 100 to begin with, then they become 101

File: endpoints.json [hunk 2]
```diff
@@ -66,5 +67,20 @@
       }
 
 		}
-	}
+	},
+  "PATCH /api/articles/:article_id": {
```
Comment (roisinoreilly) [inline endpoints.json:71]: nicely remembered

File: mvc/models/article.models.js [hunk 3]
```diff
@@ -35,4 +35,18 @@ function fetchArticles() {
     });
 }
 
-module.exports = { fetchArticlesById, fetchArticles };
+function updateVotes(article_id, inc_votes) {
+  return db
+    .query(
+      `UPDATE articles
```
Comment (roisinoreilly) [inline mvc/models/article.models.js:41]: excellent

**Review by roisinoreilly — APPROVED** [review]
I'd also just like to see a 400 test for an invalid id, and potentially an extra test to check if you can patch negative numbers (which is definitely a behaviour we want!). I'd also suggest a few other tests for if the patch object is formatted incorrectly or doesn't contain the correct keys or values. Otherwise your code itself looks excellent, really great job!
